schemas/v2/index.json

```diff
@@ -356,6 +356,64 @@
             ],
             "additionalProperties": false
         },
+        "GerritConfig": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "const": "gerrit",
+                    "description": "Gerrit Configuration"
+                },
+                "url": {
+                    "type": "string",
+                    "format": "url",
+                    "description": "The URL of the Gerrit host.",
+                    "examples": [
+                        "https://gerrit.example.com"
+                    ],
+                    "pattern": "^https?:\\/\\/[^\\s/$.?#].[^\\s]*$"
+                },
+                "projects": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "List of specific projects to sync. If not specified, all projects will be synced. Glob patterns are supported",
+                    "examples": [
+                        [
+                            "project1/repo1",
+                            "project2/**"
+                        ]
+                    ]
+                },
+                "exclude": {
+                    "type": "object",
+                    "properties": {
+                        "projects": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "examples": [
+                                [
+                                    "project1/repo1",
+                                    "project2/**"
+                                ]
+                            ],
+                            "description": "List of specific projects to exclude from syncing."
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "revisions": {
```

Comment:
nit: could we exclude this from the schema for now and then add it back once we have branch support?

Reply:
I thought from our offline dicussion that you were planning on refactoring the way branches are tags are handled. Should I plan to handle this in the future? Or should I just remove it and you will update this later?

Reply:
I'll remove this for now.